code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -534,4 +534,3 @@ meteor_act
 	var/obj/item/organ/internal/heart/heart = internal_organs_by_name[BP_HEART]
 	if(heart.pulse == PULSE_NONE)
 		death()
-	
```

Comment:
Don't remove this final line, you want a trailing newline at the end of files ideally